package.json

```diff
@@ -22,6 +22,7 @@
     "react-dom": "18.1.0",
     "react-firebase-hooks": "^5.0.3",
     "react-multi-carousel": "^2.8.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
I had thought that next.js had come in with react-router-dom by default? If I'm wrong then I'm fine with having it in here.

Reply:
Next actually has its own router: https://nextjs.org/docs/api-reference/next/router